src/Fantomas.Tests/LetBindingTests.fs

```diff
@@ -546,4 +546,10 @@ let ``multiple empty lines between equals and expression`` () =
 
 
     ()
-"""
\ No newline at end of file
+"""
+
+[<Test>]
```

Comment:
Makes more sense to have the test in AttributeTests.fs

Reply:
done